### PR TITLE
[WIP] made AsseticBundle work when templating is disabled and only Twig is enabled

### PR DESCRIPTION
**Pull request by fabpot**
I've started to see how to make the bundle compatible with a Symfony project where Twig is used without the templating sub-system. It turns out that it's going to be complex. So, I'm pushing this first step, which far from being enough. Reading through the code, I can see many weird/smart (your choice) use of some Symfony features.

So, the question is: is it worth it? Seeing the answers on Twitter, I think I won't spend too much on this: https://twitter.com/fabpot/status/648520881145253888

### CODE REVIEW

File: DependencyInjection/Compiler/TemplateResourcesPass.php [hunk 1]
```diff
@@ -29,7 +29,11 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        $engines = $container->getParameter('templating.engines');
+        if (!$container->hasParameter('templating.engines')) {
+            $engines = array('twig');
```
Comment (stof) [inline DependencyInjection/Compiler/TemplateResourcesPass.php]: we should check whether TwigBundle is enabled though

**Comment by stof** [comment]
I think supporting this use case is a good idea (it is something I have on my TODO-list, but lower priority than other things). Even though using gulp/grunt/whatever would be my recommendation today, a PHP-only alternative like Assetic may help for some people.

**Comment by shakaran** [comment]
By this commit, I have the following error:

You have requested a non-existent parameter "templating.engines".

Since I am using symfony flex, future Symfony 4, I don't know where or how set the templating.engines variable. I read that I need to set in a .env file, but I don't know if the name will be TEMPLATING_ENGINES or how

**Comment by shakaran** [comment]
Ok, I just see that it is under config/packages/framework.yaml

Adding:

```
framework:
    templating:
        engines: ['twig', 'php']
```

**Pull request closed without merging**